tomcat/src/main/java/org/apache/coyote/http11/request/HttpRequestStartLine.java

```diff
@@ -30,7 +31,16 @@ public static HttpRequestStartLine from(final BufferedReader reader) throws IOEx
 
         final String[] startLineContents = startLine.split(" ");
 
-        return new HttpRequestStartLine(startLineContents[0],
+        return new HttpRequestStartLine(HttpMethod.findMethod(startLineContents[0]),
+                takeUri(startLineContents[1]),
+                startLineContents[2],
+                takeQueryParams(startLineContents[1]));
```

Comment:
```suggestion
        final String method = startLineContents[0];
        final String uri = startLineContents[1];
        final String protocol = startLineContents[2];

        return new HttpRequestStartLine(HttpMethod.findMethod(method),
                takeUriPath(uri),
                protocol,
                takeQueryParams(uri));
```
위와 같이 변수를 사용해 startLineContents 배열의 요소의 의미를 드러내면 어떨까요?

Reply:
좋습니다!!